base/cfk-components/confluent-no-auth-tls/kustomization.yaml

```diff
@@ -1,8 +1,14 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - secrets
-  - zookeeper.yaml
+  - ../../../resources/application-secrets-rbac
+  - ../cp-version
+  - rest-class.yaml
```

Comment:
Don't think we need the rest class unless we are using RBAC? not sure though

Reply:
Hmmm... yea, I've always been somewhat confused about that.  I'll test, and remove if not required.

Reply:
Right you are.... removed it from the other RBAC example as well :+1: